runtime/syntax/vim.vim

```diff
@@ -19,14 +19,14 @@ syn cluster vimCommentGroup	contains=vimTodo,@Spell
 
 " Special and plugin vim commands {{{2
 syn match   vimCommand contained	"\<z[-+^.=]\="
-syn keyword vimOnlyCommand contained	fix[del] sh[ell] P[rint]
+syn keyword vimOnlyCommand contained fix[del] sh[ell] te[aroff] P[rint]
 syn keyword vimStdPlugin contained	DiffOrig Man N[ext] S TOhtml XMLent XMLns
 
 " Vim-specific options {{{2
-syn keyword vimOnlyOption contained	biosk bioskey cp compatible consk conskey cm cryptmethod edcompatible guipty key macatsui mzq mzquantum osfiletype oft renderoptions rop st shelltype sn shortname tenc termencoding ta textauto tx textmode tf ttyfast ttym ttymouse tbi ttybuiltin wiv weirdinvert
+syn keyword vimOnlyOption contained	biosk bioskey cp compatible consk conskey cm cryptmethod edcompatible guipty key macatsui mzq mzquantum osfiletype oft renderoptions rop st shelltype sn shortname tenc termencoding ta textauto tx textmode ttym ttymouse tbi ttybuiltin wiv weirdinvert
```

Comment:
> it is a smell

I agree, but I'm not totally sure, since we already have the compatibility options in place. If they're highlighted as red then people might think they're an error when they aren't.

The point of the compatibility shims was so that people could have `set ttyfast` in the same vimrc they use for Vim and Nvim without having to wrap it in `has('nvim')` or `exists('+ttyfast')`. With that said, I'd rather we not define it as an error until we actually remove the compatibility shim.